src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpMetricExporter.cs

```diff
@@ -106,19 +122,17 @@ public override ExportResult Export(in Batch<Metric> metrics)
 
                 try
                 {
-                    wire_size += request.CalculateSize();
-
-                    // [abeaulieu] This need to be refactored since we will need several requests now.
-                    // request.AddMetrics(this.ProcessResource, metrics);
-                    // [abeaulieu] TODO: Spread the request load?
-                    if (!this.exportClient.SendExportRequest(request))
+                    if (request == null || !this.exportClient.SendExportRequest(request))
                     {
-                        // Fail as soon as one export fails.
-                        // [abeaulieu] Should we try the rest of the messages and only return the result at the end?
-                        // Need to handle partial success here... what happens if only some payloads made it?
-                        // Ideally the telemetry should be split into individual messages that get processed using the retry mechanism/
                         return ExportResult.Failure;
                     }
+
+                    if (this.baseDelay > 0)
+                    {
+                        var random = new Random();
```

Comment:
This random should probably be `static readonly private` and reused everywhere instead of creating it on every iteration.